prebid-server/developers/pbs-build-an-analytics-adapter.md

```diff
@@ -41,21 +43,183 @@ A simple [filesystem](https://github.com/prebid/prebid-server/tree/master/analyt
 
 It can be configured with:
 
-```
+```yaml
 analytics:
   file:
     filename: "path/to/file.log
 ```
+
 Prebid Server will then write sample log messages to the file you provided.
 
+### HTTP Analytics Module in PBS-Go
+The HTTP Analytics Module can send selected Analytics Events to a specified HTTP endpoint.
+
+The sample rate has to be between `0.0` (never sample) and `1.0` (always sample). The sample rate is always evaluated and defaults to `0.0`.
+
+The module uses [github.com/antonmedv/expr](github.com/antonmedv/expr) for complex filter options. The [analytics object](https://github.com/mllrsohn/prebid-server/tree/master/analytics/core.go) is always passed into the expression.
+
+Here's how it can be configured:
+
+```yaml
+analytics:
+    http:
+        # Required: enable the module
+        enabled: true
+        endpoint: 
+            # Required: url where the endpoint post data to
+            url: "https://my-rest-endpoint.com"
+            # Required: timeout for the request (parsed as golang duration)
+            timeout: "2s"
+            # Optional: enables gzip compression for the payload
+            gzip: false
+            # Optional: additional headers send in every request
+            additional_headers:
+                X-My-header: "some-thing"
+        buffer: # Flush events when (first condition reached)
+            # Size of the buffer in bytes
+            size: "2MB" # greater than 2MB (size using SI standard eg. "44kB", "17MB")
+            count : 100 # greater than 100 events
+            timeout: "15m" # greater than 15 minutes (parsed as golang duration)
+        auction: 
+            sample_rate: 1 # sample rate 0-1.0 to sample the event, 0 (default) disables the collector for those events
+            filter: "RequestWrapper.BidRequest.App.ID == '123'" # Optional filter
+        amp:
+            sample_rate: 0.5 # 50% of the events are sampled
+            filter: "" 
+        video:
+            sample_rate: 1 # Sample rate, f 0-1 set sample rate, 1 is 100%
+            filter: "" 
+        setuid:
+            sample_rate: 0.25 # 25% of the events are sampled
+            filter: "" 
+        cookie_sync:
+            sample_rate: 0 # events are not sampled
+            filter: "" 
+        notification:
+            sample_rate: 1
+            filter: "" 
+```
+
+The payload to the endpoint is an array of the analytics object plus the `type` and the `createdAt` date.
+
+Analytics objects can be of the following types:
```

Comment:
Are /events supported? https://docs.prebid.org/prebid-server/endpoints/pbs-endpoint-event.html

Reply:
'notification'? Ok, please document that as meaning /event.

> Maybe we can Link to the analytics modules interface in github ?

Don't have a strong feeling about this. Go ahead, will see how it looks.